### PR TITLE
Add 2015 december devsprint report.

### PR DESCRIPTION
**Pull request by ChillarAnand**

### CODE REVIEW

File: _posts/2015-12-19-december-dev-sprint.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+layout: post.html
+title: December 2015 Dev Sprint report
+tags: [meetup, devsprint]
+---
+
+The [December BangPypers Dev Sprint](http://www.meetup.com/BangPypers/events/225059174/) happened at the [AdNear][https://near.co/] office in Kormanagala.
+
+Dev sprint started around 10 O Clock. The agenda was to contribute to open source projects. There were 20 participants. The event started with introduction about Python open source projects and participants.
```
Comment (kracekumar) [inline _posts/2015-12-19-december-dev-sprint.md]: There were 30 participants.

File: _posts/2015-12-19-december-dev-sprint.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Thanks for AdNear for offering the space, food, snacks.
+Thanks to mentors for helping participants with their first contributions.
+Thanks to participants for making their contributions to open source.
+
+
+We also have a [mailing list](https://mail.python.org/mailman/listinfo/bangpypers), [slack](https://bangpypers.slack.com) chanel where discussion happens about Python.
+
+
+[Arun Ravindran]: https://twitter.com/arocks
```
Comment (kracekumar) [inline _posts/2015-12-19-december-dev-sprint.md]: Remove the mention

File: _posts/2015-12-19-december-dev-sprint.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+[Sayan]: https://twitter.com/chowdhury_sayan
```
Comment (kracekumar) [inline _posts/2015-12-19-december-dev-sprint.md]: Remove the mention

File: _posts/2015-12-19-december-dev-sprint.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+[Krace]: https://twitter.com/kracetheking
+[Harsh Gupta]: https://github.com/hargup
```
Comment (kracekumar) [inline _posts/2015-12-19-december-dev-sprint.md]: Remove the mention

**Comment by ChillarAnand** [comment]
My bad. Fixed.

**Pull request merged**